docs/build-mobile.md

```diff
@@ -0,0 +1,43 @@
+# Building LibVCX on mobile
+For general overview, see https://dev.to/jakubkoci/how-to-build-a-vcx-ios-library-f32
+
+### TODO: Revise instruction below
```

Comment:
Maybe this should be a [markdown comment](https://gist.github.com/jonikarppinen/47dc8c1d7ab7e911f4c9)